content/2024-09-16_report-ze-sprintu-v-msenem.md

```diff
@@ -0,0 +1,65 @@
+Title: Report z letního sprintu v Mšeném
+Date: 2024-09-16 18:00:00
+Author: Mia Bajić
+Gravatar: miabajic.miabajic@gmail.com
+About: Mia je vývojářka, developer advocate, a spoluorganizátorka pražského Pyva, českého PyConu, pražské Python Pizzy & EuroPythonu.
+
+Jako každý rok se během léta sjíždí Pythonisti a Pythonistky ze všech koutů republiky směrem k Roudnici nad Labem, do malebné obce Mšené-lázně. Naším cílem je chalupa rodiny Vysokých, která nás už od roku 2016 hostí pro letní komunitní víkendovku.
+
+Sprint je úžasná akce, kde se všichni potkáme, přes den pracujeme na společných projektech, a večer grilujeme a zpíváme u ohně. Letos se nás sešlo kolem 20 z celé republiky.
+
+![pes]({static}/images/pes.jpg)
+
+
+### Průběh akce
+
+Ve čtvrtek dorazila menší skupina, která se pustila do příprav. Letos jsme experimentovali s jídelníčkem, abychom efektivněji nakoupili. A také jsme podpořili místní farmáře nákupem čerstvé zeleniny přímo z farmy.
+
+![pripravy]({static}/images/pripravy.jpg)
```

Comment:
Všiml jsem si, že tohle je fotka z úklidu. Ale to asi nevadí.

Reply:
Já vím, ale lepší z příprav jsem nenašla. Pokud máš nějakou lepší, klidně ji hoď. 🙂